api/app/controllers/spree/api/v2/storefront/products_controller.rb

```diff
@@ -3,8 +3,10 @@ module Api
     module V2
       module Storefront
         class ProductsController < ::Spree::Api::V2::BaseController
+          include Spree::Api::V2::Storefront::CollectionOptionsHelpers
```

Comment:
@VygovskySergey we can even move this to `Spree::Api::V2::CollectionOptionsHelpers` as we're going to reuse this also on the Platform side